<!DOCTYPE HTML>
<html>
	<head>
		<title>Generic - Editorial by HTML5 UP</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">
						<div class="inner">

							<!-- Header -->
								<header id="header">
									<a href="index.html" class="logo"><strong>CELL Team Six</strong></a>

								</header>

							<!-- Content -->
								<section>
									<header class="main">
										<h1>Ras protein superfamily</h1>

										<hr class="major" />
											<h1>H-RAS</h1>
											The H-Ras protein, a member of the Ras subfamily, is a GTPase that alternates between an active GTP-bound form and an inactive GDP-bound form.<br><br>

											Guanine nucleotide exchange factors (GEFs) are responsible for activating H-Ras by exchanging the bound GDP with GTP. GTPase activating proteins (GAPs) inactivate H-Ras by removing the γ-phosphate of the bound GTP in order to convert it to GDP. However, H-Ras also has intrinsic enzymatic hydrolysing activity and can convert its own GTP to GDP although this process is significantly slower and thus sped up by the action of GAPs. Examples of GAPs that help inactivate H-Ras are p120GAP and RASA3<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3128632/"><sup>1</sup></a>. <br><br>

											H-Ras is localized to the plasma membrane. In the activated GTP-bound state, H-Ras functions to activate downstream effectors in the early stages of signalling transduction pathways involved in cell proliferation and survival.<br><br>

											H-Ras coupling to downstream effectors is mainly dependent on conformational alterations of the Switch 1 and Switch 2 regions of the protein controlled by GTP or GDP binding. When H-Ras is bound to GTP, its binding surface has a high affinity for downstream effectors. For instance, the dissociation constant of H-Ras-GTP bound to RAF1 is 18nM<a href="https://stke.sciencemag.org/content/2004/250/re13.abstract"><sup>2</sup></a>.<br><br>

											The main downstream effectors activated by H-Ras are the protein kinase v-Raf-1 murine leukaemia viral oncogene homolog 1 (c-Raf-1) and Phosphatidylinositol 3-kinase (PI3K)<a href="https://pubmed.ncbi.nlm.nih.gov/11027944/"><sup>3</sup></a>. The c-Raf-1 pathway is associated with cell proliferation whereas the PI3K pathway is linked to prosurvival signalling<a href="https://pubmed.ncbi.nlm.nih.gov/21779497/"><sup>4</sup></a>.<br><br>

											Receptor tyrosine kinases, such as the platelet-derived growth factor receptor beta, are activated by growth factor binding. Upon this, through SH2 domains, the GRB2-Sos complex is recruited to the plasma membrane via the phosphotyrosine docking sites of the receptor. Sos, a GEF, can interact with H-Ras to exchange its GDP with GTP to activate it. H-Ras can then interact downstream with Raf kinase to activate it which in turn induces the phosphorylation of MEK and MAPK leading to cell proliferation<a href="https://pubmed.ncbi.nlm.nih.gov/27239960/"><sup>5</sup></a>. H-Ras-GTP can also activate PI3K allowing it to phosphorylate several components involved in regulating apoptosis<a href="https://pubmed.ncbi.nlm.nih.gov/21779497/"><sup>4</sup></a>.<br><br>

											Somatic mutations in RAS genes are common to 30% of all cancerous tumors<a href="https://pubmed.ncbi.nlm.nih.gov/17384584/"><sup>6</sup></a>. KRAS is the most often mutated isoform (85%), whereas NRAS (11%) and HRAS (4%) mutations are seen less frequently<a href="https://jcs.biologists.org/content/129/7/1287"><sup>7</sup></a>. Mutations of different RAS isoforms are associated with specific cancer types, with HRAS predominantly mutated in head and neck, thyroid, and bladder cancer. A missense gain-of-function mutation usually activates the oncogenic HRAS gene. Usually, changes occur at amino acid positions 12, 13, and 61, with these three mutational hotspots resulting in different HRAS functional impairments<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3354961/"><sup>8</sup></a>. The most common G12V substitution causes a decrease in GAP sensitivity, G13D mutation leads to reduced GAP-mediated GTP hydrolysis and increased intrinsic nucleotide exchange, and Q61L substitution shows decreased GTP hydrolysis, loss of GAP sensitivity, and a higher rate of nucleotide exchange<a href="https://jcs.biologists.org/content/129/7/1287"><sup>7</sup></a>. Mutated HRAS gets locked in its active GTP bound state, leading to activation of downstream pathways without extracellular stimulation. Persistent activation of RAS mediated pathways causes uncontrolled cell division, growth which leads to tumorigenesis. <br><br>

											Germline mutations in the HRAS gene are associated with the autosomal dominant condition called the Costello syndrome, characterized by coarse facial features, intellectual impairment, heart problems, and cancerous and noncancerous tumor predisposition<a href="https://pubmed.ncbi.nlm.nih.gov/22261753/"><sup>9</sup></a>. 92% of patients show a heterogenous missense point mutation in which guanine is changed into alanine in codon 12, less frequently to cytosine<a href="https://pubmed.ncbi.nlm.nih.gov/16372351/"><sup>10</sup></a>. This mutation results in the substitution of amino acid glycine with serine (G12S) in the HRAS protein. Mutated HRAS is constitutively active due to impaired GTP hydrolysis, which, like in cancer, leads to uncontrollable cell division, cell overgrowth, and tumor formation.<br><br>





<br><br>
												<h2>Other significant proteins in the family</h2>
												<h3>RRAD (GTP-binding protein RAD)</h3>
												It is highly expressed in the heart and less frequently expressed in the lungs and skeletal muscle. It is overexpressed in skeletal muscle of type II diabetes patients. RRAD is a negative regulator of glucose uptake and may be involved in the organization of the cytoskeleton. It acts as a calcium channel regulator involved in cardiac antiarrhythmia via suppressing voltage-gated L-type Ca2+ currents. It also acts as a tumor suppressor and may serve as an epigenetic biomarker in cancer.
<br><br>
												<h3>GEM (GTP-binding protein GEM)</h3>
												It is anchored to the inner side of the plasma membrane. Its specific function is still unknown, but GEM may be involved in receptor-mediated signal transduction as a regulatory protein. GAP was shown to contain guanine nucleotide-binding activity but lack intrinsic GTPase activity.
<br><br>
												<h3>K-Ras</h3>
												K-Ras is a protein encoded by the KRAS gene which is the most frequently mutated oncogene. There are two splice variants for the KRAS gene which gives rise to two forms of the K-Ras protein: K-Ras4A and K-Ras4b. These proteins differ in the structure of their C-terminal regions and thus the methods by which they traffic to the plasma membrane.
<br><br>
												<h3>K-Ras4a</h3>
												This is the form of K-Ras that is more weakly expressed in human cells. When activated it is also involved in the RAF1/MAPK signaling pathway to regulate cell proliferation with RAF1 being its downstream effector. Another effector of K-Ras4a is hexokinase 1 which is a unique effector to this protein. This interaction is oncogenic and results in increased glycolysis that feeds into tumour metabolism.
<br><br>
												<h3>K-Ras4b</h3>
												This is the dominant form of K-Ras that is highly expressed in human cells. K-Ras4b is also an upstream component of the RAF1/MAPK signalling pathway. However, it has been shown that K-Ras4b can activate RAF1 8.4-, 4.4- and 2.3- fold better than H-Ras, N-Ras and K-Ras4a respectively.
<br><br>
												<h3>N-Ras</h3>
												It is also a GTPase that transitions between an activated GTP-bound form and an inactivated GDP-form involved in the RAF1/MAPK signalling pathway. It shuttles between the Golgi apparatus and the plasma membrane. The NRAS gene is also an oncogene and when mutated it has been associated with cancers such as follicular thyroid cancer and juvenile myelomonocytic leukaemia.



<br><br>
												<h3>References</h3>
												<ol>
												<li>Grewal, T., Koese, M., Tebar, F. and Enrich, C., 2011. Differential Regulation of RasGAPs in Cancer. Genes & Cancer, 2(3), pp. 288-297.
												<li>Colicelli, J., 2004. Human RAS Superfamily Proteins and Related GTPases. Science Signaling, 2004(250), pp. re13-re13.
												<li>Matozaki, T., Nakanishi, H. and Takai, Y., 2000. Small G-protein networks:. Cellular Signalling, 12(8), pp. 515-524.
												<li>Castellano, E. and Downward, J., 2011. RAS Interaction with PI3K: More Than Just Another Effector Pathway. Genes & Cancer, 2(3), pp. 261-274.
												<li>Michael, J., Wurtzel, J. and Goldfinger, L., 2016. Regulation of H-Ras-driven MAPK signaling, transformation and tumorigenesis, but not PI3K signaling and tumor progression, by plasma membrane microdomains. Oncogenesis, 5(5), pp. e228-e228.
												<li>Schubbert, S., Shannon, K. & Bollag, G., 2007. Hyperactive Ras in developmental disorders and cancer. Nat Rev Cancer, 7, pp. 295–308.
												<li>Hobbs, G. A., Der, C. J., & Rossman, K. L., 2016. RAS isoforms and mutations in cancer at a glance. Journal of cell science, 129(7), pp. 1287–1292.
												<li>Prior, I. A., Lewis, P. D., & Mattos, C., 2012. A comprehensive survey of Ras mutations in cancer. Cancer research, 72(10), pp. 2457–2467.
												<li>Gripp, K., Lin, A., 2012. Costello syndrome: a Ras/mitogen activated protein kinase pathway syndrome (rasopathy) resulting from HRAS germline mutations. Genet Med, 14, pp. 285–292.
												<li>Estep, A. L., Tidyman, W. E., Teitell, M. A., Cotter, P. D., & Rauen, K. A., 2006. HRAS mutations in Costello syndrome: detection of constitutional activating mutations in codon 12 and 13 and loss of wild-type allele in malignancy. American journal of medical genetics. Part A, 140(1), pp. 8–16.
												</ol>















									</header>


								</section>

						</div>
					</div>

				<!-- Sidebar -->
					<div id="sidebar">
						<div class="inner">

							<!-- Menu -->
							<nav id="menu">
								<header class="major">
									<h2>Menu</h2>
								</header>
								<ul>
									<li><a href="index.html">ALK pathway</a></li>
									<li><a href="ras.html">Ras family</a></li>
									<li>
										<span class="opener">Sequence information</span>
										<ul>
											<li><a href="fasta.html">FASTA</a></li>
											<li><a href="msa.html">Multiple Sequence Alignment</a></li>
											<li><a href="blast.html">BLAST</a></li>
										</ul>
									</li>
									<li>
										<span class="opener">Structure</span>
										<ul>
											<li><a href="s_overview.html">Overview</a></li>
											<li><a href="s_loop.html">P-loop</a></li>
											<li><a href="s_switch1.html">Switch I</a></li>
											<li><a href="s_switch2.html">Switch II</a></li>
											<li><a href="s_g4g5.html">G4, G5, HVR</a></li>
										</ul>
									</li>
									<li><a href="summary.html">Summary</a></li>
								</ul>
							</nav>

							<!-- Section -->


							<!-- Section -->
							<section>
								<header class="major">
									<h2>Created by</h2>
								</header>
								Kornelija Bakutyte<br>
								Ines Fernandez Mosquera<br>
								Yasmine Metaxa<br>
								Thomas Roberts<br>
								Filip Wilk<br>
							</section>

						<!-- Footer -->
							<footer id="footer">
								<p class="copyright">&copy; All rights reserved. Design: <a href="https://html5up.net">HTML5 UP</a>.</p>
							</footer>

					</div>
				</div>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
