<!DOCTYPE HTML>
<!--
	Editorial by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Generic - Editorial by HTML5 UP</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">
						<div class="inner">

							<!-- Header -->
								<header id="header">
									<a href="index.html" class="logo"><strong>CELL Team Six</strong></a>

								</header>

							<!-- Content -->
								<section>
									<header class="main">
										<h1>P-loop</h1>
										<center><span class="Fit"><img src="images/loop_fig_0.png" alt="" width=50% length=auto/></span></center>

										<h2>Key points</h2>
										<ul>
											<li>Mutations at G12 and G13 make Ras oncogenic because they (the mutants) prevent the hydrolysis reaction.
											<li>K16 makes H-bonds with the phosphates of the bound nucleotide to stabilize it during hydrolysis.
											<li>S17 coordinates the Mg<sup>2+</sup> ion, which is involved in the GTPase reaction.
											<li>Therefore, the P-loop is fundamental for the correct binding of the nucleotide, which is necessary for GTP hydrolysis.
										</ul>

										<h2>P-loop description</h2>
										Ras proteins are formed by 2 sections: a highly conserved G domain (aa 1 -165) and a hypervariable region (aa 166-188/89).<br>

										The G domain can be subdivided into 2 lobes:
										<ul>
											<li><b>Effector lobe (aa 1-86):</b> this lobe contains the catalytic machinery which allows Ras to interact with effector protein pathways, and that is why it is perfectly conserved between all Ras isoforms1. Also due to its functional importance, mutations at some key sites are the most common oncogenic mutation sites (G12, G13 and Q61). This lobe contains 3 of the 5 highly conserved motifs<a href="https://pubmed.ncbi.nlm.nih.gov/28630043/"><sup>2</sup></a>: G1 ( P-loop), G2 (Switch I) and G3 (Switch II).
											<li><b>Allosteric lobe (aa 87 – 165):</b> Though not perfectly similar, this lobe has a sequence homology of 80-90% between the isoforms<a href="https://pubmed.ncbi.nlm.nih.gov/32241873/"><sup>1</sup></a>.  This lobe is not explicitly involved in Ras signalling, playing instead a “supporting role”: it is involved in the interactions with membranes, lipids and nucleotides<a href="https://pubmed.ncbi.nlm.nih.gov/32241873/"><sup>1</sup></a><sup>,</sup><a href="https://www.sciencedirect.com/science/article/pii/S0022283611010175"><sup>3</sup></a>.
											</ul>

									<h2>Effector Lobe</h2>

									Looking in detail at the structure of the Effector Lobe (aa 1-86):<br><br>


									<h3>P-LOOP (aa 10-17)<a href="https://pubmed.ncbi.nlm.nih.gov/28630043/"><sup>2</sup></a></h3>

									The G1 loop, also known as the phosphate binding loop (P-loop), or the Walker A motif (original name, hardly used anymore) is a fundamental part of the nucleotide-binding region of proteins, which is evidenced by the fact that is exists in both ATP- and GTP-binding proteins<a href="https://www.tandfonline.com/doi/abs/10.1080/10409238.2018.1431605?journalCode=ibmg20"><sup>4</sup></a><sup>,</sup><a href="https://doi.org/10.1016/0968-0004(90)90281-F"><sup>5</sup></a>. <br><br>

									Its function is to bind and correctly position the α- and β- phosphates of the nucleotides – which in the case of Ras are GDP and GTP. It does not interact with the ϒ-phosphate, so it does not distinguish between the di- and tri-phosphate forms. <br><br>

									The general consensus sequence for any NDP or NTP binding P-loop is, where X represents any amino acid<a href="https://pubmed.ncbi.nlm.nih.gov/28630043/"><sup>2</sup></a><sup>,</sup><a href="https://stke.sciencemag.org/content/2004/250/re13"><sup>6</sup></a><sup>,</sup></sup><a href="https://pubs.acs.org/doi/abs/10.1021/bi00233a001"><sup>7</sup></a>: <br><br>

									<center>GXXXXGK(S/T) </center><br>

									This allows us to infer the importance of the G10, G15, K16 and S/T17 amino acids. <br><br>

									Highlighting the importance of the G12 and G13 residues, the more specific consensus region for the P-loop in Ras yields<a href="http://perspectivesinmedicine.cshlp.org/content/8/8/a031427"><sup>8</sup></a>: <br><br>

									<center>GAGGVGKS</center><br><br>

									The normal P-loop is characterised for its lack of side chains – except that of K16, which is involved in binding to the phosphate. The amide groups in the backbone of the loop point towards the β-phosphate of GTP, making hydrogen bonds (H-bonds) to the phosphate oxygens and therefore fixing the orientation of the β-phosphate, which is important during GTPase activity<a href="https://www.nature.com/articles/337090a0"><sup>9</sup></a>. A metal ion, Mg<sup>2+</sup> is coordinated to the oxygen atoms of the β-phosphate<a href="https://www.nature.com/articles/337090a0"><sup>9</sup></a>. <br><br>

									We will therefore consider the consequences that mutations in all of the sites highlighted above have on the function of Ras. <br><br>

									<h3>G10</h3>

									From the Ramachandra plot for Ras it was determined that G10 (as well as G15) is in an angular conformation that is not allowed for any other amino acid<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC552258/"><sup>10</sup></a><sup>,</sup><a href="https://www.sciencedirect.com/science/article/pii/009286749090018A"><sup>11</sup></a>, which would suggest why these residues are so highly conserved in the P-loops of nucleotide binding proteins (NBP) and why the structure of these loops is conserved. <br><br>

									<center><span class="Fit"><img src="images/loop_fig_1.png" alt="" width=50% length=auto /></span></center>

									<center><b>Fig 1.</b> Structurally, the only residue that fits at position 10 is glycine.</center> <br><br>



									<h3>G12</h3>

									Mutations of this residue are one of the most studied mutations in Ras due to their oncogenic character and their high frequency of occurrence: approximately 1/3 of human cancers have altered Ras proteins1, and from those, 44 different point mutations have been characterised, with 99.2% occurring at codons 12, 13 or 6112. <br><br>

									These mutations are oncogenic because they prevent the “switching off” of the Ras molecule: Ras proteins alternate between active (GTP bound) and inactive (GDP bound) states, but mutations at this site prevent GTP hydrolysis -both intrinsically and GAP-mediated8-, leaving Ras in a constitutively active signalling state1,13. <br><br>

									Ras is involved in 2 types of GTP hydrolysis: GAP-mediated and intrinsic. Mutations in the G12 residue affect both types:
									<ul>
										<li><b>GAP-mediated hydrolysis:</b> G12 is a fundamental residue because during Ras-GAP binding, GAP introduces an arginine finger into the Ras active site to stabilize the negative charges that accumulate during a hydrolysis reaction. When the arginine finger is introduced, it is done in such a way that it forms van der Waal interactions with the Cα of G1212,14. If there is any amino acid at that position that contains a side chain that is bigger than glycine’s hydrogen atom, the interaction is displaced and the mutant will not be able to form the transition state complex with GAP12,15 (Fig 2). Even the G12A mutation, which is the sterically smallest mutation that could take place cannot form the GAP transition state due to steric clashes14,16. Nonetheless, G12 mutants can still bind to GAP with an affinity similar to that of the wild-type protein, suggesting that the arginine finger’s neighbouring residues during the Ras-GAP interaction are not as crucial for the ground state of the complex as for the transition state14,15.
										<li><b>Intrinsic hydrolysis:</b> G12 mutations impair the active site conformation of Ras, resulting (except for in the case of G12P) in a 10-fold reduction of the intrinsic hydrolysis rate in vitro12.
									</ul> <br><br>

									<center><span class="Fit"><img src="images/loop_fig_2.png" alt="" width=50% length=auto/></span></center>

									<center><b>Fig 2.</b> Shows the point of contact between the arginine finger (R789) and G12. Any side chain at position 12 would push the P-loop and R789 away and would not allow Van der Waals interactions to take place.</center><br><br>

									The importance of G12 as a residue is also highlighted by the fact that most RAS family members contain a glycine at position 12 except ERAS, RASD1/2 and DIRAS3, which are RAS proteins which are constitutively active4; they do not act as molecular switches and do not change between GDP and GTP bound states.<br><br>

									Unlike what is observed for G10, the Ramachandran angles of the wild-type H-Ras structure have values allowed for any amino acid (φ = -60⁰, ϕ = +132⁰), so any mutation can take place at this location without leading to a major rearrangement of the protein7,10,11. Indeed, even mutants like G12R where the mutated residue’s side chain is much larger than glycine’s, do not significantly alter the structure of the loop11; instead, the main effect of side chains at this position is to physically  block access to the catalytic site7, which affects the functioning of the protein.<br><br>

									Although any amino acid residue is allowed at this position, the mutations of the Ras isoforms preferentially occur at different codons12:
									<ul>
										<li><b>H-Ras:</b> 50% of its mutations occur at the 12th position, and 40% at the 13th.
											<li><b>K-Ras:</b> 80% of its mutations occur at the 12th position.
												<li><b>N-Ras:</b> 35% of mutations occur at the 12th position, and 60% at the 61st.
									</ul><br><br>

									Considering that these 3 oncogenic mutations take place in a region that is perfectly conserved between the three isoforms, their differences in codon specificity are surprising. Further unexpected preferences at these positions are found in the single DNA base substitutions that determine the amino acid mutations. In the 12th codon:<br><br>
									<ul>
										<li><b>H-Ras:</b> favours G12V mutations. However, more generally, mutations at this codon are 3 times more likely to be a transversion (pyrimidine base is changed to a purine or vice versa) than a transition (purine to purine or pyrimidine to pyrimidine) compared to the other Ras isoforms12.<br><br>
											<li><b>K-Ras:</b> 43% of its mutations involve a Guanine to Adenine transition at the second base of the codon12, resulting in G12D mutations:
												<ul>
													<li>
													Glycine = GGU/GGC/GGA/GGG
													<li>
													Aspartic acid  =   G<span style="color: red">A</span>U/G<span style="color: red">A</span>C
													</ul>
													The remaining 57% of mutations involve a Guanine to Thymine transversion at the second base of the codon12 (uracil in the mRNA), resulting in G12V mutations:
													<ul>
													<li>Glycine   =   GGU/GGC/GGA/GGG
													<li>Valine     =   G<span style="color: red">U</span>U/G<span style="color: red">U</span>C/G<span style="color: red">U</span>A/G<span style="color: red">U</span>G
													</ul>
													<li> <b>N-Ras:</b> favours similar types of mutations as K-Ras, though at lower rates12.
													</ul>


					It has been theorised that the heterogeneity can be due to variations in the DNA’s primary sequence, which would reflect why K-Ras and N-Ras have mutations of the same type since they share the exact DNA sequence that encodes G12 (and Q61)12. Another plausible explanation for these preferences are the differences in membrane-interacting components12, which are found exclusively in the allosteric lobe and the C-terminal HVR, which are sections of the protein that do differ between isoforms.<br><br>

					While the abovementioned mutations are the most common, the most highly transforming in G12 mutants are: G12I, G12L, G12T, G12R and G12V. G12I/L/T mutants require more than one base substitution to occur, explaining their relative rarity. G12R, though it only requires a single substitution at the loci of the 1st base, is also rare. G12V combined with G12D make up the bulk of the observed mutations.12<br><br>

					Based on all of this, we will consider in detail the G12P, G12V, G12D and G12R mutants.<br><br>

					<h3>G12P</h3>
					As mentioned, G12P mutants are the only G12 mutants that do not make Ras oncogenic. G12P mutants are not oncogenic because they show a slight decrease in the dissociation rate of GDP and a slight increase in the dissociation constant for GTP17. This suggests that compared to the wild-type Ras, G12P mutants are more likely to be found bound to GDP.<br><br>

					The presence of a pyrrolidine ring (from the proline residue) is responsible for the weaker affinity towards GAP and for the failure to form the Ras-GAP transition state that leads to hydrolysis17.<br><br>

					<h3>G12V</h3>

					G12V mutants show a lower affinity to GAP than the wild-type, but they are nonetheless still able to form the Ras-GAP complex18. However, even though they can bind, G12V mutants are not activated by GAP, so they show no measurable GAP-mediated hydrolysis16.<br><br>

					The constitutively active Ras(G12V) proteins promote a relatively minor increase in Mitogen Activated Protein Kinase (MAPK) and phosphorylation in cells. In other members of the RAS superfamily, this same mutation can have different effects8.<br><br>

					The presence of valine’s side chain at position 12 also leads to steric hindrance with atoms from Switch II (aa. 60-76); in order to reduce this hindrance, the A59, G60 and Q61 residues are shifted11. This shifting means that G60 can no longer form a H-bond between its main chain NH and one of the ϒ-phosphate oxygens, so instead it forms a H-bond from its carbonyl atom to the side chain of Y9611.<br><br>

					These rearrangements lead to V12’s hydrophobic branched side chain coming into close contact with the negatively charged phosphate group11. This is an unfavourable interaction, which is relieved by slightly changing the geometry of the P-loop i.e., the P-loop of the G12V mutant becomes bigger than that of the wild-type9.<br><br>

					Because the P-loop “opens up”, two H-bonds that would normally form between the amide (NH) groups of the backbones of codons 12 and 13 to the β-phosphate are lost9. The loss of these H-bonds means that the orientation of the β-phosphate can change, either when presented to the attacking group, or as the leaving group after the ϒ-phosphate is attacked9. Regardless, the change in the orientation of the β-phosphate leads to a change in the hydrolysis rate of GTP.<br><br>

					<center><span class="Fit"><img src="images/loop_fig_3.png" alt="" width=50% length=auto/></span></center>
					<center><b>Fig 3.</b> Overlap of the P-loop of wt-HRas and Ras(G12V). PDB codes 4EFL and 4EFM respectively.</center><br><br>

					<h3>G12D</h3>

					G12D mutants show a drastic increase in the dissociation rate for GDP and a reduced GTPase activity -both intrinsic and GAP-mediated17, meaning that these mutants are usually found bound to GTP. Even though GAP-binding does not increase the rate of GTP hydrolysis, these mutants still have the same affinity for GAP as the wild-type protein17.<br><br>

					In all G12 mutants except G12D, the connection that goes from the active site to the helix α5 through a network of H-bonds is severed due to the absence of a water molecule that normally bridges Y32, Q61 and the ϒ-phosphate. Instead, this means that the H-bond forms directly between Y32 and the ϒ-phosphate 8,13 and that the connection between switch I (Y32) and the helix nucleotide residues associated with membrane orientation -found in the allosteric lobe- is severed13.<br><br>

					<center><span class="Fit"><img src="images/loop_fig_4.png" alt="" width=50% length=auto/></span></center>
					<center><b>Fig 4.</b> Some key polar interactions from Ras(G12V) as a representative of all non-G12D mutants of Ras. A) The network of H-bonds from the active site to helix α5 is severed. B) Y32 is associated with the ϒ-phosphate of GTP (PDB code 4EFM).</center><br><br>

					G12D mutants, however, find a replacement to that lost water molecule bridge. Upon crystallization G12D shows 2 possible structures:
					<ul>
						<li>In one of them, the water molecule is displaced and replaced by the side chain of the D12 residue: its side chain carboxyl group13 forms H-bonds with Y32 and the ϒ-phosphate, connecting them12. The hindrance at the active site means that the side chain of Q61 is rotated towards the solvent, which leaves the network from the allosteric side to R68 (helix α3) intact, but the connection to Q61 is impaired13.
							<li>In the other, the water molecule is present, but D12 replaces Q61 in its interaction with the water molecule. This moves the N-terminus of switch II out of place, completely severing the connectivity to the active site8,13.
							</ul><br><br>

					<center><span class="Fit"><img src="images/loop_fig_5.png" alt="" width=50% length=auto/></span></center>
					<center><b>Fig 5.</b> Direct polar interaction between the side chain of D12 with Y32 and the ϒ-phosphate in Ras(G12D) mutant. (PDB code 6JZ0)</center><br><br>

					These connections mean that in G12D mutants – unlike in the others- the water-mediated network spanning from the ϒ-phosphate to the α5 helix sensing residues is intact, which may contribute to the effectiveness and frequency of this mutation in human cancers13.

					<h3>G12R</h3>

					In G12R mutants, the guanidinium group from arginine’s side chain is placed in front of GTP’s ϒ-phosphate blocking physical access to the ϒ-phosphate while being involved in a H-bond with one of its oxygens11. In order to do this, R12 must push Q61 into the solvent13, which reorientates residues 62-65 from Switch II. By binding to GTP, the guanidium is taking the place of the bridging water molecule, which is believed may act as a nucleophile (donates an electron pair) in GTP hydrolysis11, preventing it from taking place.<br><br>

					Arginine’s side chain is also close to the carbonyl oxygen of T35, so it blocks the exit for the release from the nucleotide binding pocket, explaining why the nucleotide dissociation is slow in this mutant11,18.<br><br>

					<center><span class="Fit"><img src="images/loop_fig_6.png" alt="" width=50% length=auto/></span></center>

					<center><b>Fig 6.</b> Representation of the interactions as described above in K-Ras(G12R) mutants –crystal structure for this mutant in H-Ras was not available in the PDB; but the representation is still accurate because the effector lobe is identical in all Ras isoforms. A) The H-bond between R12’s side chain and the GTP is visible (dotted red line). Q61 is pushed into the solvent and Switch II (cyan) is disordered, which is why its full structure is not shown. B) Spherical representation of R12 and T35 to illustrate that the nucleotide binding pocket’s exit is blocked. (PDB code 6cu6)</center><br>

					<h3>G13</h3>
					Mutations in G13, like in G12, lead to the constitutive activation of Ras by making the mutant GAP-insensitive8,13, which is why these mutations are common in human cancers4,5. The same heterogeneity that was observed for G12 in terms of isoform preferences for mutations at certain codons is also observed for G1312.<br><br>

					Unlike what was observed in the Ramachandran plot of G12, the G13 angle is has unusual bonding angles (φ = 78⁰, ϕ = 11⁰)7,10,11, which are believed to be outside the angle regions allowed for even glycine18. This unusual angular conformation means that in the case of mutation of the G13 residue, the local structure of the P-loop is likely to be perturbed7,18.<br><br>

					In the normal conformation i.e., glycine at position 13, there is a H-bond between the backbone amide of G13 and the oxygen atom bridging the β- and ϒ-phosphates of GTP10.<br><br>

					The literature on the biochemical and structural biology of G13 is scant, but it is clear from the structure of the RasGAP complex that any side chain at this position would clash to some extent with the arginine finger in GAP and/or the tyrosine at position 32 (Y32) which is involved in intrinsic hydrolysis at the active site12.<br><br>

					<center><span class="Fit"><img src="images/loop_fig_7.png" alt="" width=50% length=auto/></span></center>
					<center><b>Fig 7.</b> The close proximity between G13 and the nucleotide does not allow for a side chain larger than glycine’s. Any side chain at the 13th position would also clash with Y32 (PDB code 4rsg) </center> <br><br>


					Despite this, it has been found that amino acids with small side chains e.g., G13A and G13V can form the transition state with GAP, but larger ones, such as G13R cannot12. It has been concluded that codon 13 is not as sensitive to steric hindrance as codon 12, but it is sensitive to charge16. This is because all the main chain amide groups from residues 13-18 are positioned such that they are pointing towards the GTP’s phosphate group, creating a positively polarised local electrostatic field10,18.<br><br>

					Therefore, the mutation of G13 into other small non-polar residues like G13A and G13V are not transforming (oncogenic)18. It has been observed that G13A and G13V mutants bind to GAP with wild-type affinity and that G13A hydrolysis GTP almost as fast as wild-type Ras16. In G12V mutants, however, hydrolysis is reduced 2000-fold, but nonetheless, the GTPase activity is still measureable16. This, combined with the fact that G13D and G13R show no GTPase activity16, is why it is believed that for a G13 mutation to be oncogenic, there must be a change in size and charge.<br><br>

					<h3>G15</h3>

					Codon 15, like codon 10, is in a conformation which is not allowed for any amino acid other than glycine due to the unusual φ and ϕ angles at these positions10,11. A mutation in this site is likely to lead to the disappearance of the structure at the P-loop, rendering the protein inoperative.<br><br>

					<h3>K16</h3>

					Position 16 is not a site for oncogenic mutations, meaning that altering the amino acid residue will not make the Ras protein function abnormally5.<br><br>

					Despite not being a transforming mutation, K16 is one of 22 highly conserved sites in Ras7. This is because it forms H-bonds to the main chain oxygens of other residues of the P-loop - more specifically, residues 10 (G10) and 11 (A11) 5,7 (Fig 8) -, as well as the β- and ϒ- phosphates of the bound NTP5,7,10.<br><br>

					The importance of K16 in the binding of the phosphates means that if there is a mutation at this site, the affinity for the guanine nucleotide is drastically reduced, as happens in the K16N mutant5.<br><br>

					A characteristic feature of lysine as part of the P-loop is that is the only amino acid residue that has a side chain9 – the other residues are predominantly G and A which have virtually no side chain. This longer side chain makes K16 is essential for binding9,10 -and thus, for catalysis- of the GTP, as well as other parts of the structure of Ras such as the β strands 1 (at V8 and V9) and 3 (at D56, T58) and G60 residue in Switch II7.<br><br>

					<center><span class="Fit"><img src="images/loop_fig_8.png" alt="" width=50% length=auto/></span></center>
					<center><b>Fig 8.</b> Polar interactions between K16 and G10 and A11 as seen from three different angles. C) also shows the interactions between K16 and the β- and ϒ- phosphates.</center>

					<h3>S17</h3>

					Residue 17 can be either serine (S17) or threonine (T17), but since in Ras the conserved residue is S17, we will not consider the case where threonine is found at this position. It is also worth mentioning that there is debate about whether codon 17 is part of the P-loop or the α-helix that immediately follows the loop, but we will consider it to be part of the loop since the Carr-Purcell- Meiboom-Gill measurements place it in this region19.<br><br>

					S17 plays a key role in allowing GTP binding by coordinating the magnesium ion (Mg<sup>2+</sup>)5. These divalent cations play such an instrumental role in the activity of the Ras protein, that the difference in affinity between Ras and GDP/GTP in the presence or absence of Mg<sup>2+</sup> is greater than 500-fold20. The possible functions of Mg<sup>2+</sup> may include shielding of the negative charge associated with hydrolysis, increasing the acid strength of the leaving group (β-phosphate), activation of the nucleophile and/or stabilization of the transition state of the hydrolysis reaction8.10.<br><br>

					S17 coordinates the action of Mg<sup>2+</sup> by forming H-bonds with the ion from their -OH group as one of the interactions in which <sup>Mg2+</sup> is involved in. The coordination octahedral sphere around Mg<sup>2+</sup> is made from interaction with 6 oxygens20:
<br>
					<ul>
						<li>In the GDP bound form10,18,20:
							<ul>
							<li>OH group from S17
								<li>β-phosphate from GDP
								<li>4 water molecules clustered around the protein.
								</ul>
								<li>In the GTP bound form8,10,18,20:
									<ul>
								<li>OH group from S17
								<li>OH group from T35 (in Switch II)
								<li>Β-phosphate from GTP
								<li>ϒ-phosphate from GTP
								<li>2 water molecules
								</ul>
								</ul>

								<center><span class="Fit"><img src="images/loop_fig_9.png" alt="" width=50% length=auto/></span></center>
								<center><b>Fig 9.</b> Interactions between Mg<sup>2+</sup> (black) and its binding in the A) GTP (PDB code 4rsg) and B) GDP (PDB code 6ZL3) bound states.</center><br><br>

								Mutations are codon 17 are not oncogenic, in fact, some of its mutants (S17N, S17A and S17C) have been shown to be negative inhibitors for downstream signalling4,20, that is, the mutation decreases the affinity for GTP, but not GDP, and so, they have been found to inhibit the proliferation of cells by interfering with the functions of wild-type Ras proteins20.<br><br>

								The most common mutants are:<br>

								<ul>
								<li>S17N
									<ul>
								<li>This mutation is commonly used as a dominant negative Ras mutant because overexpressed Ras with this mutation binds tightly to RasGEFs and sequesters them from endogenous Ras proteins4. Therefore, it interferes with Ras activation.
</ul>
								<li>S17A
<ul>
		<li>						S17A are the mutants with the greatest effect on the interaction between Ras and the guanine nucleotide because they cause an increase in the dissociation rate of GTP20. This increased dissociation constant increases the affinity of Ras for GDP by more than 400-fold, which means the rate of binding to GDP is 27 times greater than that of GTP even if there is a higher concentration of GTP than GDP20.
</ul>
								</ul>

								This mutation also increases the dissociation constant for Mg<sup>2+</sup> (by 30-fold), which, combined with the lack of GTPase activity in these mutants suggests that the GTPase reaction is completely dependent on the presence of a bivalent metal ion20.
<br><br>

<h3>References</h3>
<ol>
<li>“A historic perspective and overview of H-Ras structure, oncogenicity and targeting (2020) DOI: 10.1158/1535-7163.MCT-19-0660: https://pubmed.ncbi.nlm.nih.gov/32241873/ <br>
<li>“The small GTPases K-Ras, N-Ras and H-Ras have distinct biochemical properties determined by allosteric effects” DOI 10.1074/jbc.M117.778886 https://pubmed.ncbi.nlm.nih.gov/28630043/ <br>
<li>“Analysis of binding site hot spots on the surface of Ras GTPase” doi:10.1016/j.jmb.2011.09.011: https://www.sciencedirect.com/science/article/pii/S0022283611010175<br>
<li>“Structural fingerprints, interactions and signal networks of RAS family proteins beyond RAS isoforms” (2018) DOI: 10.1080/10409238.2018.1431605 https://www.tandfonline.com/doi/abs/10.1080/10409238.2018.1431605?journalCode=ibmg20<br>
<li>“The P-loop: a common motif in ATP- and GTP-binding proteins” (1990) https://doi.org/10.1016/0968-0004(90)90281-F: <br>
<li>“Human RAS Superfamily Proteins and Related GTPases” doi:10.1126/stke.2502004re13: https://stke.sciencemag.org/content/2004/250/re13<br>
<li>“The ras Protein family: evolutionary tree and role of conserved amino acids” (1991) https://pubs.acs.org/doi/abs/10.1021/bi00233a001<br>
<li>“The K-Ras, N-Ras and H-Ras Isoforms: unique conformational preferences and implications for targeting oncogenic mutants” (2018) doi: 10.1101/cshperspect.a031427: http://perspectivesinmedicine.cshlp.org/content/8/8/a031427<br>
<li>“Structural differences between a Ras oncogene protein and the normal protein” (1989)  https://www.nature.com/articles/337090a0<br>
<li>Refined crystal structure of the triphosphate conformation of H-Ras p21 at 1.35 A resolution: implications for the mechanism of GTP hydrolysis” (1990) https://www.ncbi.nlm.nih.gov/pmc/articles/PMC552258/<br>
<li>“3D structures of H-Ras p21 mutants: molecular basis for their inability to function as signal switch molecules” (1990) doi: 10.1016/0092-8674(90)90018-a: https://www.sciencedirect.com/science/article/pii/009286749090018A<br>
<li>“A comprehensive survey of Ras mutations in cancer” (2012) doi:10.1158/0008-5472.CAN-11-2612 https://pubmed.ncbi.nlm.nih.gov/22589270/<br>
<li>“Direct attack on RAS: intramolecular communication and mutation-specific effects (2015) DOI: 10.1158/1078-0432.CCR-14-2148: https://pubmed.ncbi.nlm.nih.gov/25878362/<br>
<li>“The Ras-RasGAP complex: structural basis for GTPase activation and its loss in oncogenic Ras Mutants” (1997) DOI: 10.1126/science.277.5324.333 https://pubmed.ncbi.nlm.nih.gov/9219684/<br>
<li>“GTPase activating proteins: structural and functional insights 18 years after discovery” (2005) DOI 10.1007/s00018-005-5136-x: https://link.springer.com/article/10.1007/s00018-005-5136-x<br>
<li>“Fluoride complexes of oncogenic Ras mutants to study the Ras-RasGAP interaction” (2008) . DOI 10.1515/BC.2008.132 https://pubmed.ncbi.nlm.nih.gov/18713003/<br>
<li>“3D structures and Properties of a Transforming and a non-transforming G12 mutant of p21H-Ras” (1993) https://doi.org/10.1021/bi00084a005<br>
<li>“structure of the guanine-nucleotide-binding domain of the Ha-Ras oncogene product p21 in the triphosphate conformation” (1989) https://www.nature.com/articles/341209a0<br>
<li>“Solution structure of the state 1 conformer of GTP-bound H-Ras Protein and Distinct Dynamic Properties between the State 1 and State 2 Conformers” (2011) , DOI 10.1074/jbc.M111.227074 https://www.sciencedirect.com/science/article/pii/S0021925820506212<br>
<li>“Kinetic and structural analysis of the Mg2+-binding site of the guanine nucleotide-binding protein p21” (1993) https://www.jbc.org/article/S0021-9258(18)54022-9/pdf<br>
</ol>



									</header>


								</section>

						</div>
					</div>

				<!-- Sidebar -->
					<div id="sidebar">
						<div class="inner">

							<!-- Search -->


							<!-- Menu -->
							<nav id="menu">
								<header class="major">
									<h2>Menu</h2>
								</header>
								<ul>
									<li><a href="index.html">ALK pathway</a></li>
									<li><a href="ras.html">Ras family</a></li>
									<li>
										<span class="opener">Sequence information</span>
										<ul>
											<li><a href="fasta.html">FASTA</a></li>
											<li><a href="msa.html">Multiple Sequence Alignment</a></li>
											<li><a href="blast.html">BLAST</a></li>
										</ul>
									</li>
									<li>
										<span class="opener">Structure</span>
										<ul>
											<li><a href="s_overview.html">Overview</a></li>
											<li><a href="s_loop.html">P-loop</a></li>
											<li><a href="s_switch1.html">Switch I</a></li>
											<li><a href="s_switch2.html">Switch II</a></li>
											<li><a href="s_g4g5.html">G4, G5, HVR</a></li>
										</ul>
									</li>
									<li><a href="summary.html">Summary</a></li>
								</ul>
							</nav>

							<!-- Section -->


							<!-- Section -->
							<section>
								<header class="major">
									<h2>Created by</h2>
								</header>
								<p>CELL Team Six</p>
								<ul class="contact">
									<li class="icon solid fa-envelope"><a href="#">information@untitled.tld</a></li>
								</ul>
							</section>

						<!-- Footer -->
							<footer id="footer">
								<p class="copyright">&copy; All rights reserved. Design: <a href="https://html5up.net">HTML5 UP</a>.</p>
							</footer>

					</div>
				</div>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
