<!DOCTYPE HTML>
<!--
	Editorial by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Generic - Editorial by HTML5 UP</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">
						<div class="inner">

							<!-- Header -->
								<header id="header">
									<a href="index.html" class="logo"><strong>CELL Team Six</strong></a>

								</header>

							<!-- Content -->
								<section>
									<header class="main">
										<h1>Switch II</h1>
										<center><span class="Fit"><img src="images/switch2_fig_0.png" alt="" width=50% length=auto /></span></center>


										<h2>Key points</h2>
										<ul>
											<li>G60 is conserved in all GTP binding proteins, because it interacts (along with T35 from switch I) with the ϒ-phosphate of GTP to create a “loaded spring mechanism”.
											<li>Q61 is a highly conserved residue in the whole RAS superfamily because it is involved in the hydrolysis of the nucleotide triphosphate (it is also present in ATP binding proteins).
											<li>Mutations at Q61 prevent the hydrolysis reaction and make Ras oncogenic.
										</ul>

										<h2>Switch II description</h2>

										Switch II, also called the G3 loop, is a region which is generally said to span from G60 to E761,2,3 in H-Ras, such that it is made up of loop L4 and the α2 helix4,5. Some of the amino acids in this regions are conserved because switch II is involved in characterising the ATP or GTP binding sites in many proteins; however, differences in this structure are what allow the various nucleotide binding proteins to be part of their own signalling pathways despite the similarities in the structure between all of them.
										Despite these differences, all RAS superfamily proteins appear to have a conserved <sup>57</sup>DXXG<sup>60</sup> motif6,7, and a conserved glutamine at position 61 (Q61).These characteristics can be seen in our <a href="www.cellteamsix.github.io/msa.html">Multiple Sequence Alignment</a>.

										<br><br>
										As such, we will divide our consideration of the switch II region into 2 sections:
										 <ul>
											 <li>DXXG motif
												 <li>Q61 to E76 residues
												 </ul>

										<h3><sup>57</sup>DXXG<sup>60</sup> motif</h3>
										The fundamental residue in this motif for the correct functioning of nucleotide binding proteins is the glycine at position 60. This codon, along with T35 from switch I is responsible for binding the ϒ-phosphate of the nucleotide through its backbone amide group5,8,9, giving rise to a “loaded spring mechanism”7. Release of this mechanism by hydrolysis of the H-bonds formed between T35 and G60 with the ϒ-phosphate relaxes both switch regions (I and II) into the GDP-bound conformations7.<br><br><br><br>



										<center><span class="Fit"><img src="images/switch2_fig_1.jpg" alt="" width=50% length=auto /></span></center>
										<center><b>Fig 1.</b>Hydrogen bonds from the T35 and G60 residues to the ϒ-phosphate hold switch I and II in a high energy conformation i.e., “loaded spring mechanism”. Image obtained from here [hyperlink the “here” to reference 7]
</center><br><br>

										The DXXG motif is not only conserved amongst member of the RAS superfamily, but it is also a common motif among ATP-binding proteins7, where G60 is also involved in the binding of the ϒ-phosphate through its main chain amide.
										The importance of G60 is such to the correct functioning of Ras, that even though the remaining residues from the motif are involved in Mg2+ coordination6, which is not one of the main functions of the switch II region, on occasion, switch II has been described as extending from D57 to E766 to account highlight the presence of the DXXG motif at that position and to not remove G60 from the residues which make up this domain.

										<h3>Q61 to E76 residues</h3>
										From this selection of residues, the only codon that is conserved in all nucleotide binding proteins is Q61, the other codons accept other residues to allow proteins to have different functions. Note, however, that the whole switch II regions is conserved between the Ras isoforms (H-Ras, K-Ras and N-Ras).<br><br>
										The glutamine residue needs to be conserved at position 61 because its mutation makes Ras constitutively active2,10 i.e., oncogenic; in most mutants of Q61 there is at least a 10-fold decrease in the rate of intrinsic hydrolysis in vitro11,12. In fact, its prevalence in cancers is such, that in 95% of mutated N-Ras cancers, the affected residue is Q6111. In the case of H-Ras, there is a 40:50 split between codons 12 and 61 respectively11. Mutations at the 61st position have been linked predominately to skin and thyroid cancers11.<br><br>
										The reason why glutamine is necessary at that position is because it is the only residue capable of establishing H-bonds to the active site and the nucleophilic water molecule simultaneously (Fig 2.), which are connections needed to stabilise the transition state of Ras13 (transition between GDP and GTP bound states). Even a mutation to its most closely related amino acid, asparagine (N), which only lacks one carbon atom in its side chain in relation to glutamine, would not be able to successfully participate in both interactions because it would not be able to reach far enough into the active site11.

										<br><br><br><br>
										<center><span class="Fit"><img src="images/switch2_fig_2.png" alt="" width=50% length=auto /></span></center>
										<center><b>Fig 2.</b> Interactions between Q61 (red) from Switch II with G12 (orange) from the active site. GTP is shown as a dotted structure in black.</center>
										<br><br>

										The details of the interactions between Q61 and its binding partners depend on whether GAP-mediated or intrinsic hydrolysis is taking place.
										<ul>
											<li>In GAP-mediated hydrolysis, the amide group of the Q61 side chain forms one H-bond to GAP’s arginine finger10,13 at the active site. The other H-bond is established from its side chain carbonyl group with the nucleophilic water molecule11.
												<li>In intrinsic hydrolysis, the active site connection is formed directly with the ϒ-phosphate, and the interaction with the nucleophilic water occurs in the same manner as in the previous scenario.12,13
												</ul>
										Glutamine’s ability to form both of these interactions in the Ras-GTP bound state, leads to two different conformations of switch II6, depending on whether the diphosphate or triphosphate nucleotide is bound:
										<ul>
										<li>R state: Catalytically active
										<li>T state: Catalytically inactive
										</ul>
										In the catalytically incompetent T state, switch II is found in a disordered state1,14 because the helix α3 clashes with switch II6. It is worth noting that when switch II is in this conformation, the allosteric binding site in the allosteric lobe is empty15.


										<br><br><br><br>
										<center><span class="Fit"><img src="images/switch2_fig_3.png" alt="" width=50% length=auto /></span></center>
										<center><b>Fig 3.</b> In the T-state, the switch II loop (red) is disordered, and there is no ligand in the allosteric binding site (dark blue). (PDB code 2RGE)</center>
										<br><br>

										The catalytically active R state arises from the ordering of the switch II region as a result of the binding of a ligand in the allosteric binding site (or the binding of GAP to the active site in GAP-mediated hydrolysis). The binding of these molecules to the corresponding binding site, causes a shift in the helix α3 and loop L73,14, driving them away from switch II11 and towards the helix α46,14, which allows this region of the protein to become ordered2,3 into a fully helical conformation6,15 (Fig 4). This disorder to order transition of switch II culminates in the placement of Q61 at the active site2,3, where it may form the interactions with the ϒ-phosphate and catalytic water molecule1.




										<br><br><br><br>
										<center><span class="Fit"><img src="images/switch2_fig_4.png" alt="" width=50% length=auto /></span></center>
										<center><b>Fig 4.</b> In the R-state, there is a ligand bound to the allosteric binding site, and switch II is ordered into a helix. (PDB code 3k8y)</center>
										<br><br>

										These interactions allow Q61 to perform a critical role in the hydrolysis of GTP, such that its mutation leads, as mentioned above, to constitutively active oncogenic mutants. One of the most common and highly activating mutants is:
										<h3>Q61L</h3>
										This mutant arises from a missense mutation in the DNA sequence since:<br><br>
										<center>Glutamine =  C<span style="color: red">A</span>A / C<span style="color: red">A</span>G</center>
										<center>Leucine   =  C<span style="color: red">T</span>A / C<span style="color: red">T</span>G  (among others)</center>
<br>


										Ras(Q61L) binds GAP with an affinity similar to that of the wild-type, but like it happened at the other oncogenic sites (G12 and G13, which are part of the P-loop [Filip, could you put a hyperlink to the P-loop tab at this point?]), the mutant is not capable of forming the transition state11,13, and hydrolysis does not increase as a result of the interaction.
										Most Q61 mutants (Q61L among them) displace the nucleophilic water molecule2 at the active site, forming instead a H-bond directly from L61 to Y32 and the ϒ-phosphate11. Therefore, these mutants stabilise an anti-catalytic conformation of the active site2, reducing the rate of hydrolysis6,16. Additionally, the direct interaction that forms between L61 and Y32 breaks the connection between the active site and the α5 helix network15.

										The remaining residues in switch II seem to contribute to the stabilisation of switch II6 to allow Q61 to be involved in catalysis. One codon that is worth highlighting is R68, which is an ordered residue6 in switch II despite the region’s general disordered structure.
										In order for the transition between states T and R to occur14, R68 must be part of an interlobal binding pocket called Cluster Q3, along with residue Y96 from the allosteric lobe. Interaction of ligands with this binding pocket (some of GAP’s residues are amongst these ligands) facilitate the transition and allow the positioning of Q61 at the active site3. Some of the residues found at the Cluster Q site include those at positions 90-95, which are characterised by being “hot spots” for Ras isoform-specific differences6, allowing switch II to be involved in determining which signalling pathways may be activated by which particular Ras protein.













<br><br>
<h3>References</h3>
<ol>
										<li>“Allosteric modulation of Ras positions Q61 for a direct role in catalysis” (2010) https://www.pnas.org/content/107/11/4931:
										<li>“Allosteric modulation of Ras-GTP is linked to signal transduction through RAF kinase” (2010) DOI 10.1074/jbc.M110.193854: https://pubmed.ncbi.nlm.nih.gov/21098031/
										<li>“Analysis of binding site hot spots on the surface of Ras GTPase” (2011) doi:10.1016/j.jmb.2011.09.011: https://www.sciencedirect.com/science/article/pii/S0022283611010175
										<li>“Molecular dynamics simulations of the Hras-GTP complex and the HRas-GDP complex” (2013) DOI: 10.1002/qua.24457: https://onlinelibrary.wiley.com/doi/abs/10.1002/qua.24457
										<li>“Molecular switch for signal transduction: structural differences between active and inactive forms of protooncogenic ras proteins” (1990) DOI: 10.1126/science.2406906: https://pubmed.ncbi.nlm.nih.gov/2406906/

										<li>“The small GTPases K-Ras, N-Ras and H-Ras have distinct biochemical properties determined by allosteric effects” (2017) , DOI 10.1074/jbc.M117.778886: https://pubmed.ncbi.nlm.nih.gov/28630043/

										<li>“The guanine nucleotide binding switch in 3D”(2001) DOI: 10.1126/science.1062023. https://science.sciencemag.org/content/294/5545/1299?hwshib2=authn%3A1616406442%3A20210321%253Ae41e4aed-5683-435d-96b6-650836afe9d0%3A0%3A0%3A0%3AoChellUz4YuRkbrxvctWBw%3D%3D
										<li>“Structural Fingerprints, interactions and signaling networks of RAS family proteins beyond RAS isoforms” (2018) https://doi.org/10.1080/10409238.2018.1431605:
										<li>“Transformation efficiency of Q61 Ras mutants linked to structural features of the switch regions” (2007) doi: 10.1016/j.str.2007.10.011.: https://pubmed.ncbi.nlm.nih.gov/18073111/

										<li>“Structural Fingerprints, interactions and signaling networks of RAS family proteins beyond RAS isoforms” (2018) https://doi.org/10.1080/10409238.2018.1431605:

										<li>“A comprehensive survey of Ras mutations in cancer” (2012) doi:10.1158/0008-5472: https://pubmed.ncbi.nlm.nih.gov/22589270/
										<li>“Quantitative measurement of intrinsic GTP hydrolysis in carcinogenic Q61 mutants in H-Ras” (2018) DOI: 10.1021/acs.biochem.8b00878 https://pubs.acs.org/doi/abs/10.1021/acs.biochem.8b00878
										<li>“Fluoride complexes of oncogenic Ras mutants to study the Ras-RasGAP interacton”(2008) DOI 10.1515/BC.2008.132: https://www.degruyter.com/document/doi/10.1515/BC.2008.132/html
										<li>Direct attack on RAS: intramolecular communications and mutation-specific effects” doi: 10.1158/1078-0432.CCR-14-2148: https://clincancerres.aacrjournals.org/content/21/8/1810.abstract
										<li>“The K-Ras, N-Ras and H-Ras isoforms: unique conformational preferences and implications for targeting oncogenic mutants (2018) doi: 10.1101/cshperspect.a031427: https://pubmed.ncbi.nlm.nih.gov/29038336/
										<li>RAS isoforms and mutations in cancer at a glance” (2016) doi:10.1242/jcs.182873: https://pubmed.ncbi.nlm.nih.gov/26985062/
</ol>






									</header>


								</section>

						</div>
					</div>

				<!-- Sidebar -->
					<div id="sidebar">
						<div class="inner">

							<!-- Search -->
								<section id="search" class="alt">
									<form method="post" action="#">
										<input type="text" name="query" id="query" placeholder="Search" />
									</form>
								</section>

							<!-- Menu -->
							<nav id="menu">
								<header class="major">
									<h2>Menu</h2>
								</header>
								<ul>
									<li><a href="index.html">ALK pathway</a></li>
									<li><a href="ras.html">Ras family</a></li>
									<li>
										<span class="opener">Sequence information</span>
										<ul>
											<li><a href="fasta.html">FASTA</a></li>
											<li><a href="msa.html">Multiple Sequence Alignment</a></li>
											<li><a href="blast.html">BLAST</a></li>
										</ul>
									</li>
									<li>
										<span class="opener">Structure</span>
										<ul>
											<li><a href="s_overview.html">Overview</a></li>
											<li><a href="s_loop.html">P-loop</a></li>
											<li><a href="s_switch1.html">Switch I</a></li>
											<li><a href="s_switch2.html">Switch II</a></li>
											<li><a href="s_g4g5.html">G4, G5, HVR</a></li>
										</ul>
									</li>
									<li><a href="summary.html">Summary</a></li>
								</ul>
							</nav>

							<!-- Section -->


							<!-- Section -->
							<section>
								<header class="major">
									<h2>Created by</h2>
								</header>
								<p>CELL Team Six</p>
								<ul class="contact">
									<li class="icon solid fa-envelope"><a href="#">information@untitled.tld</a></li>
								</ul>
							</section>

						<!-- Footer -->
							<footer id="footer">
								<p class="copyright">&copy; All rights reserved. Design: <a href="https://html5up.net">HTML5 UP</a>.</p>
							</footer>

					</div>
				</div>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
